File: modules/expo-sign-camera/android/src/main/java/expo/modules/signcamera/ExpoSignCameraModule.kt
package expo.modules.signcamera

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExpoSignCameraModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoSignCamera")
    View(ExpoSignCameraView::class) {
      Events("onLandmarks", "onCameraError")
      Prop("hand") { view: ExpoSignCameraView, hand: String -> view.hand = hand }
    }
  }
}

File: modules/expo-sign-camera/ios/ExpoSignCameraModule.swift
import ExpoModulesCore

public class ExpoSignCameraModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExpoSignCamera")
    View(ExpoSignCameraView.self) {
      Events("onLandmarks", "onCameraError")
      Prop("hand") { (view: ExpoSignCameraView, hand: String) in view.hand = hand }
    }
  }
}
